variables.tf

```diff
@@ -58,26 +58,18 @@ variable "roles" {
     granted_to_roles          = optional(list(string))
     granted_to_database_roles = optional(list(string))
     granted_database_roles    = optional(list(string))
-    database_grants = optional(object({
-      all_privileges    = optional(bool)
-      with_grant_option = optional(bool, false)
-      privileges        = optional(list(string), null)
-    }))
     schema_grants = optional(list(object({
       all_privileges             = optional(bool)
       with_grant_option          = optional(bool, false)
       privileges                 = optional(list(string), null)
-      all_schemas_in_database    = optional(bool, false)
       future_schemas_in_database = optional(bool, false)
```

Comment:
shouldn't we remove this one as well? you removed it from locals.tf

Reply:
Done